anxcloud/schema_lbaas.go

```diff
@@ -0,0 +1,18 @@
+package anxcloud
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func schemaLBaaSLoadbalancer() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "LoadBalancer name.",
+		},
+		"ip_address": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "LoadBalancer IP address.",
```

Comment:
something like "Address of the Engine management API of the LoadBalancer" is probably better, also, this does not have to be an IP address as far as I'm aware, but could also be a DNS name

Reply:
then again "ip_address" matches the API name of that attribute, so fine with that